modules/qna/src/views/full/index.tsx

```diff
@@ -23,12 +23,12 @@ const QnAList: FC<Props> = ({
   refreshQnaCount
 }) => {
   const [flows, setFlows] = useState([])
-  const [filterContexts, setFilterContexts] = useState([])
   const [questionSearch, setQuestionSearch] = useState('')
-  const [showImportModal, setShowImportModal] = useState(false)
   const [currentTab, setCurrentTab] = useState('qna')
   const [currentLang, setCurrentLang] = useState(contentLang)
   const [url, setUrl] = useState('')
+  const [filterOptions, setFilterOptions] = useState({ disabled: true, incomplete: true, active: true })
+  const [sortOption, setSortOption] = useState(+1)
```

Comment:
Can you put this in the reducer instead?  Also, the sort option feels weird. Can you use a string value instead ? +1 / -1 doesn't mean anything